Classes/Repository/RepositoryCodeDirectoryViewController.swift

```diff
@@ -99,28 +99,10 @@ final class RepositoryCodeDirectoryViewController: UIViewController, UITableView
     }
 
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath)
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath) as! StyledTableCell
```

Comment:
Eeeeep force-unwrap 💥☠️🚫 Can you handle this w/out forcing? Either `fatalError` or
```swift
if let cell = cell as? StyledTableCell {
  cell.setup(with: file)
}
```

Reply:
using a if let would not work, because we can't return `nil`, so I decided to use `fatalError`